src/api/transaction/utils.js

```diff
@@ -54,11 +58,13 @@ function prepareTransaction(transaction: any, remote: any, instructions: any,
 
   function prepareMaxLedgerVersion(callback_) {
     if (instructions.maxLedgerVersion !== undefined) {
-      txJSON.LastLedgerSequence = parseInt(instructions.maxLedgerVersion, 10);
+      txJSON.LastLedgerSequence =
+        typeof instructions.maxLedgerVersion === undefined ?
```

Comment:
My error, it intended to be `typeof instructions.maxLedgerVersion === 'undefined'`
strange that flow didn't catch this :disappointed: